### ISSUE TITLE
Ability to disable default filters

### ISSUE THREAD

**Issue by fosamax** [OpenEmu/OpenEmu#1767]
Not really an issue but I wish we could select which filters/shaders to use with OpenEmu.

Currently, I can add shaders to OpenEmu, and also remove shaders I imported myself but I can't remove shaders that I don't use at all if they are from the default ones.

My main concern here is performance related since I own a pretty old Mac mini (late 2009), with a Nvidia 9400 (256 Mo shared memory) and some shaders are too heavy to handle for that hardware

When testing Zelda ALTTP (Higan core but Snes9x gives the same results here), I can use any of the included shaders except those : Beam4, CRT, NTSC, SABR

Maybe CRT could be used without barrel distortion or with less demanding cores.

There's also Blinky which I have no use for.

So maybe and option to enable/disable filter the same way we can enable/disable core could be usefull for people with older hardware.

I had good results with CRT Hyllian LQ even if it's a bit blurry.

Any idea for great shaders that don't cost too much on performance ?

Thanks

**Comment by clobber** [comment]
I run a machine with the same GPU as you. Anyway, just don't use shaders that can't run well on your GPU. Not going to add clutter to the UI just to 'hide' ones the 9400M can't handle.

**Comment by fosamax** [comment]
Thanks for your answer. Anyway, I found meanwhile it was possible to show package content of OpenEmu.app and then remove unwanted filters.

I guess I may eventually screw something and I will probably need to do that again after any update but I'm currently happy with that.

BTW, I tried the following filters with success :
- CRT Hyllian LQ with sharpness enabled
- CRT Hyllian with edited parameters (disabled anti ringing, dilatation and phosphor).

Really nice emulator hub.

Thanks
